awx/main/scheduler/kubernetes.py

```diff
@@ -51,6 +52,72 @@ def list_active_jobs(self, instance_group):
 
         return pods
 
+    @classmethod
+    def create_secret(self, job):
+        task = collections.namedtuple('Task', 'id instance_group')(id='', instance_group=job.instance_group)
+        pm = PodManager(task)
+        registry_cred = job.execution_environment.credential
+        host = registry_cred.get_input('host').split('/')[0]
+        username = registry_cred.get_input("username")
+        password = registry_cred.get_input("password")
+
+        # Construct container auth dict and base64 encode it
+        token = b64encode("{}:{}".format(username, password).encode('ascii')).decode()
```

Comment:
```
>>> "hello_🐉".encode('ascii')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character '\U0001f409' in position 6: ordinal not in range(128)
```
I don't think we want to `.encode('ascii')`. We wouldn't support unicode characters in the username/password then.

Reply:
:thinking: we'll probably want to change this line too then.  https://github.com/ansible/awx/blob/devel/awx/main/tasks.py#L845

Reply:
@Zokormazo can we add an integration test for this?